### PR TITLE
SDK-1064 update setting entities

### PR DESCRIPTION
**Pull request by sergeyspryker**
https://spryker.atlassian.net/browse/SDK-1064

### CODE REVIEW

File: src/Entity/SettingInterface.php [hunk 1]
```diff
@@ -109,6 +109,17 @@ public function getSettingType(): string;
...
+    /**
+     * Specification:
+     * - Returns `true` if a value should be set by a user in any case.
+     * - Returns `false` for default behavior.
+     *
+     * @api
+     *
+     * @return bool
+     */
+    public function isForceAskValue(): bool;
```
Comment (vol4onok) [inline src/Entity/SettingInterface.php:121]: maybe isRequire() ? if the setting is required we need to ask it.

File: src/Entity/SettingInterface.php [hunk 2]
```diff
@@ -150,4 +161,15 @@ public function isProject(): bool;
...
+    /**
+     * Specification:
+     * - Returns `true` if it just created.
+     * - Returns `false` if it already has existed.
+     *
+     * @api
+     *
+     * @return bool
+     */
+    public function isFirstInitialization(): bool;
```
Comment (vol4onok) [inline src/Entity/SettingInterface.php]: is it an important method? I'm not sure that we need it in contracts.

**Review by vol4onok — CHANGES_REQUESTED** [review]

File: src/Entity/SettingInterface.php (hunk 1 above)
Comment (sergeyspryker) [reply src/Entity/SettingInterface.php:121]: It was discussed with @pavelmaksimov25. Actually we have all the settings required. If you set null value in settings.yaml this value will be asked in any cases. But if you set some default values the question will be skipped.

File: src/Entity/SettingInterface.php (hunk 2 above)
Comment (sergeyspryker) [reply src/Entity/SettingInterface.php]: Yes i need to identify the first setting initialization to ask the user's email. In the second run it should be skipped. Usually `id` is used for that but we don't expose it

Comment (vol4onok) [reply src/Entity/SettingInterface.php]: How you are going to replace this email? in this case it's not possible

Comment (vol4onok) [reply src/Entity/SettingInterface.php]: if you need it move to interface in core like `SprykerSdk\Sdk\Core\Domain\Entity\ContextInterface`

**Review by vol4onok — APPROVED** [review]

File: src/Enum/ExecutionEnv.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+namespace SprykerSdk\SdkContracts\Enum;
+
+final class ExecutionEnv
+{
+    /**
+     * Developer execution environment name.
+     *
+     * @var string
+     */
+    public const DEVELOPER = 'developer';
```
Comment (pavelmaksimov25) [inline src/Enum/ExecutionEnv.php]: `DEVELOPER` => `DEVELOP` or `DEVELOPMENT`

**Review by pavelmaksimov25 — APPROVED** [review]

**Review by pavelmaksimov25 — APPROVED** [review]

**Pull request merged**